qrisk/stats.py

```diff
@@ -532,8 +532,9 @@ def information_ratio(returns, factor_returns):
     return np.mean(active_return) / tracking_error
 
 
-def alpha_beta(returns, factor_returns, risk_free=0.0):
-    """Calculates both alpha and beta.
+def alpha_beta(returns, factor_returns, risk_free=0.0, period=DAILY,
+               annualization=None):
```

Comment:
Does it matter that even in a US equities only world, there aren't always
252 trading days in a year?
On Jul 8, 2016 19:02, "Ana Ruelas" <notifications@github.com> wrote:

> In qrisk/stats.py
> <https://github.com/quantopian/qrisk/pull/7#discussion_r70153038>:
>
> > @@ -532,8 +532,9 @@ def information_ratio(returns, factor_returns):
> >      return np.mean(active_return) / tracking_error
> >
> >
> > -def alpha_beta(returns, factor_returns, risk_free=0.0):
> > -    """Calculates both alpha and beta.
> > +def alpha_beta(returns, factor_returns, risk_free=0.0, period=DAILY,
> > +               annualization=None):
>
> The default factor is 252 as it assumes daily returns.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/quantopian/qrisk/pull/7/files/04df4a337ca3a4cd9f62725ed3d229ec0c86953f#r70153038>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAdwxt-iO83s__U-jOYlrYuBoQ-69fYQks5qTtb4gaJpZM4JHVR3>
> .
>